src/lib/CanvasWorld.js

```diff
@@ -50,6 +55,60 @@ export default class CanvasWorld {
     return this.canvases.map(canvas => canvas.id).indexOf(canvasTarget);
   }
 
+  /** @private */
+  indexOfImageResource(imageId) {
```

Comment:
I think I'm seeing an issue where `imageId` can be undefined in the console logs. Maybe its where I added the level 0 support in adding the non-tiled images resources here: https://github.com/ProjectMirador/mirador/blob/master/src/components/OpenSeadragonViewer.js#L223

Trying to think out loud, maybe we need to change how we do the level 0 support?

Reply:
Do you happen to have an example manifest?

Reply:
This one (level 0) https://github.com/IIIF/api/blob/master/source/api/presentation/2.1/example/fixtures/1/manifest.json 

But i'm seeing it also in https://prtd.app/hamilton/manifest.json